src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-use-before-define */
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import {StyleSheet, Text, View} from 'react-native';
+import {FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `FlatList` and `Keyboard` is added in this line. It seems to be used for handling the list of users and dismissing the keyboard when scrolling. This change looks good as it enhances the user experience by providing a smoother interaction.
```diff
- import {StyleSheet, Text, View} from 'react-native';
+ import {FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->